modules/ROOT/pages/data-bridge.adoc

```diff
@@ -8,133 +8,122 @@ The data bridge is the secure connection from Anypoint API Community Manager to
-+
-Replace the access token in the authorization header with the `access_token` value.
-+
-Replace `[YourOrgDomain]` with your Salesforce organization domain name. Find the domain name by logging in to your Salesforce organization, clicking *Setup*, and clicking the section *My Domain*.
-+
-----
-    curl -X POST \
-      https://anypoint.mulesoft.com/accounts/api/connectedApplications \
-      -H 'Content-Type: application/json' \
-      -H 'Accept: */*' \
-      -H 'Accept-Encoding: gzip, deflate' \
-      -H 'Authorization: Bearer 5242XXXX-XXXX-XXXX-XXXX-XXXXXXXXfec7' \
-      -H 'Host:anypoint.mulesoft.com' \
-      -d '{"client_name":"ACM Integration App","grant_types":["authorization_code","refresh_token"],"redirect_uris":["https://[YourOrgDomain].my.salesforce.com/services/authcallback/Anypoint"],"scopes":["full","offline_access"],"public_keys":[]}'
-----
-+
-. The `cURL` command returns a response like this:
-+
-----
-    {
-      "client_id": "5fafXXXXXXXXXXXXXXXXXXXXXXXX29c9",
-      "client_secret": "9509XXXXXXXXXXXXXXXXXXXXXXXXC10E",
-      "client_name": "ACM Integration App",
-      "redirect_uris": [
-        "https://[YourOrgDomain].my.salesforce.com/services/authcallback/Anypoint"
-      ],
-      "grant_types": [
-        "authorization_code",
-        "refresh_token"
-      ],
-      "public_keys": [],
-      "scopes": [
-        "offline_access",
-        "full"
-      ],
-      "enabled": true,
-      "owner_org_id": "f377XXXX-XXXX-XXXX-XXXX-XXXXXXXX9d08",
-      "as_id": "anypoint"
-    }
-----
-+
-. Note the `client_id` and `client_secret` values.
+If you already have Anypoint Identity Provider, get identity provider data from your current configuration:
 
-== Create an Authentication Provider in Your Salesforce Organization [[create-auth-provider]]
+. Click *Access Management* > *Identity Providers*.
+. On *SAML 2.0* click *Edit*.
+. Copy the issuer and the audience so you can use them in the following steps.
 
-In the following task steps, if your Anypoint Platform organization is hosted in the EU, replace all instances of `+https://anypoint.mulesoft.com+` with `+https://eu1.anypoint.mulesoft.com+`.
+== Configure Named Credential
 
-. In *Setup*, search for *Auth. Providers* using the *Quick Find* box.
-. Click *Auth. Providers* -> *New*.
-. In *Provider type*, select *Open ID Connect*.
-. Set *Name* to `Anypoint`.
-. Set *Consumer Key* to the application ID of the connected app (which you took note of when you created the app in Anypoint Platform).
-+
-This ID was returned by the Anypoint UI at the end of the task <<create-a-connected-app-with-the-user-interface>> or returned by the Anypoint API call at the end of the task <<create-a-connected-app-with-an-api-call>>.
-+
-. Set *Consumer Secret* to the application secret of the connected app created in Anypoint Platform.
+. Click *Setup* > *Named Credentials*.
+. Modify the named credential *anypoint_mulesoft*.
```

Comment:
Modify it how? 
Clarify user verb (select, create, per above edits) and use the proper format, also per above.